.redhat-ci.yml

```diff
@@ -0,0 +1,25 @@
+branches:
+    - master
+    - auto
+    - try
+
+container:
+    image: jlebon/ostree-tester:rhci
+
+tests:
+    - sh autogen.sh
+        --prefix=/usr
+        --libdir=/usr/lib64
+        --enable-installed-tests
+        --enable-gtk-doc
+    - make -j2
```

Comment:
Not many cores eh? :smile:  Down the line this seems like something we'd want to inherit from the system, but fine for now.